array-manipulation-1/src/last.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable no-unused-vars */
+function last(array) {
+  for (var i = 0; i < array.length; i++) {
```

Comment:
This loop is unnecessary, since you are returning a value immediately in its code block 👍